### PR TITLE
Adopt existing namespaces into OpenRCT2 namespace

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
A long-term refactor is to eventually restructure the codebase into proper C++ namespaces, building a tree with the `OpenRCT2` namespace at the root. Each of the components get their own logical namespace, following the folder structure and unit files.

The current namespaces originate in a time where most of the codebase was in C, not C++. This PR aims to kickstart the refactoring process by placing the existing namespaces in the `OpenRCT2` one where that was not yet the case.

In order to achieve this, this PR adds `using namespace OpenRCT2` to compilation units where appropriate, as a means to get the codebase to compile until these units have been placed in a namespace of their own.

### CODE REVIEW

File: src/openrct2/Game.h [hunk 1]
```diff
@@ -14,6 +14,8 @@
 #include <memory>
 #include <string>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/Game.h:17]: Shouldn't do this.

File: src/openrct2/entity/Peep.h [hunk 2]
```diff
@@ -26,11 +26,13 @@ constexpr uint8_t kPeepMaxEnergyTarget = 255; // Oddly, this differs from max en
 
 constexpr auto PEEP_CLEARANCE_HEIGHT = 4 * COORDS_Z_STEP;
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/entity/Peep.h:29]: Shouldn't do this in a header file.

File: src/openrct2/interface/Window.h [hunk 3]
```diff
@@ -29,6 +29,8 @@
 #include <utility>
 #include <variant>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/interface/Window.h:32]: Shouldn't do this in a header file.

File: src/openrct2/localisation/Formatter.h [hunk 4]
```diff
@@ -18,6 +18,8 @@
 #include <array>
 #include <cstring>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/localisation/Formatter.h:21]: Shouldn't do this in a header file.

File: src/openrct2/management/Research.h [hunk 5]
```diff
@@ -17,6 +17,8 @@
 
 #include <optional>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/management/Research.h:20]: Shouldn't do this in a header file.

File: src/openrct2/network/network.h [hunk 6]
```diff
@@ -24,10 +24,12 @@ constexpr const char* kNetworkLanBroadcastMsg = "openrct2.server.query";
 constexpr const char* kMasterServerURL = "https://servers.openrct2.io";
 constexpr uint16_t kMaxServerDescriptionLength = 256;
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/network/network.h:27]: Shouldn't do this in a header file.

File: src/openrct2/rct12/RCT12.h [hunk 7]
```diff
@@ -21,11 +21,13 @@
 #include <string_view>
 #include <vector>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/rct12/RCT12.h:24]: Shouldn't do this in a header file.

File: src/openrct2/ride/Ride.h [hunk 8]
```diff
@@ -29,6 +29,8 @@
 #include <memory>
 #include <string_view>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/ride/Ride.h:32]: Shouldn't do this in a header file.

File: src/openrct2/ride/RideEntry.h [hunk 9]
```diff
@@ -18,6 +18,8 @@
 
 #include <cstdint>
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/ride/RideEntry.h:21]: Shouldn't do this in a header file.

File: src/openrct2/scenario/Scenario.h [hunk 10]
```diff
@@ -20,6 +20,8 @@
 #include "../world/Map.h"
 #include "../world/MapAnimation.h"
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/scenario/Scenario.h:23]: Shouldn't do this in a header file.

File: src/openrct2/sprites.h [hunk 11]
```diff
@@ -12,6 +12,8 @@
 #include "drawing/ScrollingText.h"
 #include "rct1/Limits.h"
 
+using namespace OpenRCT2;
```
Comment (AaronVanGeffen) [inline src/openrct2/sprites.h:15]: Shouldn't do this in a header file.

**Review by Gymnasiast — APPROVED** [review]

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**